src/world_interface/real_world_interface.cpp

```diff
@@ -242,10 +243,6 @@ DataPoint<points_t> readLidarScan() {
 	return lidar::readLidar();
 }
 
-DataPoint<double> readIMUHeading() {
-	return {}; // TODO: interface with IMU
-}
```

Comment:
This doesn't have to be addressed in this PR but I'm just leaving this as a todo. This will probably break the GPS=USB build, since there won't be a `readIMUHeading()` method. We should add a dummy imu method, the same way we have a dummy gps method.

This isn't relevant right now though, so let's address that later.